app/organisation/rest.py

```diff
@@ -206,7 +206,8 @@ def get_organisation_services_usage(organisation_id):
 
 @organisation_blueprint.route("/<uuid:organisation_id>/users/<uuid:user_id>", methods=["POST"])
 def add_user_to_organisation(organisation_id, user_id):
-    new_org_user = dao_add_user_to_organisation(organisation_id, user_id)
+    permissions = request.get_json()["permissions"]
```

Comment:
minor optional - does it make sense to do the transformation here rather than in the dao method (which can then just have type `permissions: list[str]`)

```suggestion
    permissions = [p["permission"] for p in request.get_json()["permissions"]]
```

Reply:
I might change the dao method to `list[OrganisationUserPermissionType]` - I'd prefer to keep the stricter typing info until the last possible moment. But good point in general.